ui/lib/sync/addon/utils/api-method-resolver.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { capitalize } from '@ember/string';
+
+import type { DestinationType } from 'vault/sync';
+
```

Comment:
I know, I'm a documentation queen, but it is very helpful when debugging or updating. Can we add some documentation here, similar to [this](https://github.com/hashicorp/vault/blob/main/ui/app/utils/kv-path.ts#L6) file?

Reply:
I have a couple comments on lines 12 and 13 but I can expand on that. They definitely rely on implied knowledge of the endpoints.